packages/@divvi/mobile/src/viem/preparedTransactionSerialization.ts

```diff
@@ -68,3 +70,31 @@ export function getPreparedTransactions(
 ): TransactionRequest[] {
   return preparedTransactions.map(getPreparedTransaction)
 }
+
+export type SerializablePreparedTransactionsPossible = Pick<
+  PreparedTransactionsPossible,
+  'type'
+> & {
+  transactions: SerializableTransactionRequest[]
+  feeCurrency: SerializedTokenBalance
+}
+
+export function getSerializablePossibleTransaction(
```

Comment:
```suggestion
export function getSerializablePreparedTransactionsPossible(
```

naming is hard, what do you think about not making this abstraction and instead just passing serialised prepared transactions and serialised fee currency in the nav params instead?

Reply:
Was wondering about that too. I guess if we'd do that, we could pass the feeCurrency tokenId instead?
And not need much more serialization / deserialization.
However this doesn't work if the problem is around using `getFeeCurrencyAndAmounts` which needs `PreparedTransactionsResult`?

Reply:
Yes, that's the primary reason why I've decided to serialize the whole result.

Considering that we only pass possible tx one other approach I can think of is to create another function that will only calculate fees for the possible tx. It should be a really small function (and reuse it in `getFeeCurrencyAndAmounts`?) and it can omit reliance of the whole `preparedTransactionResult`.

Reply:
oh i see, never mind then 🙈 it doesn't seem worth the extra effort, it's good to leave it like this